JournalCoreData/View Controllers/EntryDetailViewController.swift

```diff
@@ -74,8 +74,8 @@ class EntryDetailViewController: UIViewController {
     
     var entryController: EntryController?
     
-    @IBOutlet weak var moodSegmentedControl: UISegmentedControl!
-    @IBOutlet weak var titleTextField: UITextField!
-    @IBOutlet weak var bodyTextView: UITextView!
+    @IBOutlet weak var moodSegmentedControl: UISegmentedControl?
+    @IBOutlet weak var titleTextField: UITextField?
+    @IBOutlet weak var bodyTextView: UITextView?
```

Comment:
You should leave these as explicitly unwrapped, since they are outlets, and should be available as soon as the view itself is loaded.

Reply:
Also, you don't need the weak keyword for IBOutlets anymore. Apple changed their mind again.